vllm/lora/ops/bgmv_expand.py

```diff
@@ -108,6 +107,7 @@ def bgmv_expand(
         override_config (Optional[Dict[str, int]], optional): Defaults to None. 
             Triton grid config
     """
+    override_config: Optional[Dict[str, int]] = None
```

Comment:
cc @jeejeelee what's the purpose of this code?

Reply:
This is the configuration passed when tuning kernel. We currently don't support tuning kernel for punica,  if tuning is implemented later, we can consider adding this parameter then.